src/app/ReadClient.cpp

```diff
@@ -1159,7 +1201,7 @@ CHIP_ERROR ReadClient::SendSubscribeRequestImpl(const ReadPrepareParams & aReadP
         mReadPrepareParams.mSessionHolder = aReadPrepareParams.mSessionHolder;
     }
 
-    mIsPeerLIT = aReadPrepareParams.mIsPeerLIT;
+    mPeerICD = aReadPrepareParams.mPeerICD;
```

Comment:
Sounds like the naming in the read prepare params needs fixing too?

Reply:
this change is removed, and unnecessary.